src/.editorconfig

```diff
@@ -0,0 +1,10 @@
+; What is EditorConfig? http://editorconfig.org/
```

Comment:
Remove and .gitignore this file

Reply:
this is great thing :)
it sets editor properties, for ex. i do use tabs for spacing, and it is configured in my VS, this file allows solution wide configuration without need to reconfigure VS properties

Reply:
Done